### DISCUSSION TITLE
Question regarding Twist/Tilt angle

### DISCUSSION THREAD

**Discussion by briffodf** [mtex-toolbox/mtex#1300] [Ask Anything] [answered]
Hello everyone ,

I am trying to calculate the twist/twilt angles of some subgrain boundaries following the tutorial and I am slightly confused about how to interpret the results. I have attached the ebsd map and a stererographic projection of the basal plane normals (circles) and basal slip direction (squares) for the two large grains in the center of the EBSD (the material is an hcp titanium alloy)

From the projection it is apparent that the subGB is due to a rotation around the c-axis and the angle between the boundary trace and the misorientation axis is near 90°. The documentation indicates that it is potentially a tilt boundary (https://mtex-toolbox.github.io/TiltAndTwistBoundaries.html):

"In the case of 2d EBSD data one usually has not full boundary information, but only the trace of the boundary with the measurement surface. Hence, it is impossible to distinguish tilt and twist boundaries. However, for twist boundaries the trace must be always perpendicular to the trace of the boundary as the trace is always perpendicular to the boundary normal. This can be easily checked from our EBSD data and allows us to exclude certain boundaries to be tilt boundaries. To do so we colorize in the following plot all subgrain boundaries according to the angle between the boundary trace and the misorientation axis. Red subgrain boundaries indicate potential tilt boundaries while blue subgrain boundaries are for sure no tilt boundaries."

But shouldn't it be a twist boundary as it is close to 90°?

Thanks in advance for your help,

Kind Regards,

Fabien

![ebsd](https://user-images.githubusercontent.com/95682723/145003840-41cb1ee8-3f1f-46c0-8a77-93180e570f34.png)
![PF](https://user-images.githubusercontent.com/95682723/145003869-80cd82f0-51d3-4c5b-ac77-58ccf25d776d.png)
ts

**Comment by kilir** [comment, marked as answer]
Hi,
I tried to correct and clarify the documentation (52d24faf6111c0c864044ea236a354b5a35a9220), and yes, it might be a twist boundary, assuming the misorientation axis is within the plane and at a high angle to your boundary trace.

Cheers,
Rüdiger

**Reply by briffodf** [reply]
Dear Rüdiger,

Thank you very much for your fast answer, it clarified my doubt.

Kind Regards,

Fabien

**Comment by zmichels** [comment]
Hi,

Someone recently wrote me about the current version of the Tilt and Twist Boundaries example mentioned in this thread. As I was looking at it today, the final plot uses the angle between the 3D misorientation axis and the subgrain boundary trace *direction*/azimuth... but it seems like it would make more sense to use the angle between the misorientation axis and the cross between the boundary direction and the vector3d.Z (?). Then red colors would indicate misorientation axes that are near the trace (i.e., possibly in the boundary plane and related to edge dislocations), and blue colors would result where misorientation axes are far from the plane defining the trace, and not likely be related to edge dislocations. Perhaps I am missing something. 

I am thinking of a plot like the following instead:
```
figure,
plot(grains.boundary,'linewidth',2)
hold on
plot(subGB,angle(cross(subGB.direction,vector3d.Z),axS)./degree,'linewidth',2)
mtexColorMap blue2red
mtexColorbar
```
![SGBs colored by angle to trace plane (rather than trace direction)](https://user-images.githubusercontent.com/8769619/146589851-4581a4de-3480-4287-96e8-e5c5620aec4f.png)


Zach

**Reply by kilir** [reply]
Hi Zach,
if I'm not completely wrong
```
angle( cross( cross( axS,vector3d.Z),vector3d.Z) ,subGB.direction)
```
would be the angle between the projection of the misorientation axis into the plane and the trace of the grain boundary.

Under the assumption that we have no idea about the grain boundary true 'dip' except that we will not sample a boundary which is parallel to our plane of observation, I'd say the only situation we can be sure for pure tilt, is that the boundary is parallel to the misorientation axis. For twist I'd say, we can never be sure except we know something about the true boundary dip.

Cheers,
Rüdiger
